src/main/java/org/hiero/sketch/table/FullMembership.java

```diff
@@ -25,6 +29,24 @@ public int getSize() {
 
     @Override
     public IRowIterator getIterator() {
-        return null;
+        return new FullMemebershipIterator(this.rowCount);
+    }
+
+    private static class FullMemebershipIterator implements IRowIterator {
+        private int cursor = 0;
+        private final int range;
+
+        public  FullMemebershipIterator(int range) {
+            this.range = range;
+        }
+
+        @Override
+        public int getNextRow() {
+            if (cursor < range) {
```

Comment:
Maybe we should setup IntelliJ to transform this automatically into this.cursor and this.range...